Makefile

```diff
@@ -263,6 +263,18 @@ snapshot-with-signing: ## Build snapshot release binaries and packages (with dum
 	# remove the keychain with the trusted self-signed cert automatically
 	.github/scripts/apple-signing/cleanup.sh
 
+snapshot-docker-assets: # Build snapshot images of docker images that will be published on release
+	$(call title,Building snapshot docker release assets)
+
+	# create a config with the dist dir overridden
+	echo "dist: $(DISTDIR)" > $(TEMPDIR)/goreleaser.yaml
+	cat .github/.goreleaser_docker.yaml >> $(TEMPDIR)/goreleaser.yaml
```

Comment:
Why does there need to be a different directory than the default when this is running on its own runner?

Reply:
I wanted to keep the makefile consistent. 

We're setting the path for a consumed `goreleaser.yaml` as `$(TEMPDIR)/gorelease.yaml` for the other make commands that invoke `goreleaser`. 

Reading it from `.github/.goreleaser_docker.yaml` would also be fine, just different from how we approach setting up the config in other commands.